core/synchronizer_integration_test.go

```diff
@@ -1066,3 +1069,88 @@ func TestSynchronizer_TC11(t *testing.T) {
 	// tear down
 	tearDown(td)
 }
+
+// TestSynchronizer_TC12: Reconciliation flow
```

Comment:
did you update the document with that TC12?

Reply:
Which document?